### PR TITLE
Dependencies bump up

### PR DESCRIPTION
**Pull request by s-pogoda**
- Bumps [json5](https://github.com/json5/json5) from 2.2.1 to 2.2.3. https://github.com/pipedrive/client-nodejs/pull/413
- Updates qs from 6.9.3 to 6.11.0 https://github.com/pipedrive/client-nodejs/pull/400
Updates formidable from 2.0.1 to 2.1.1
- Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2. https://github.com/pipedrive/client-nodejs/pull/399
- Updates minimatch from 3.0.4 to 3.1.2 https://github.com/pipedrive/client-nodejs/pull/397
Updates mocha from 8.4.0 to 10.1.0

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/012bf507-7160-48ba-8614-93238df62ea1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ The file format of it is based on [Keep a Changelog](http://keepachangelog.com/e
 For public Changelog covering all changes done to Pipedrive’s API, webhooks and app extensions platforms, see [public Changelog](https://pipedrive.readme.io/docs/changelog) with discussion area in [Developers Community](https://devcommunity.pipedrive.com/c/documentation/changelog/19).
 
 ## [Unreleased]
+- Bumped up library dependencies:
+  - Updated `json5` from 2.2.1 to 2.2.3
+  - Updated `qs` from 6.9.3 to 6.11.0
+  - Updated `formidable` from 2.0.1 to 2.1.1
+  - Updated `decode-uri-component` from 0.2.0 to 0.2.2
+  - Updated `minimatch` from 3.0.4 to 3.1.2
+  - Updated `mocha` from 8.4.0 to 10.1.0
```
Comment (RuTsEST) [inline CHANGELOG.md:16]:
```suggestion
### Updated dependencies
  - Updated `json5` from 2.2.1 to 2.2.3
  - Updated `qs` from 6.9.3 to 6.11.0
  - Updated `formidable` from 2.0.1 to 2.1.1
  - Updated `decode-uri-component` from 0.2.0 to 0.2.2
  - Updated `minimatch` from 3.0.4 to 3.1.2
  - Updated `mocha` from 8.4.0 to 10.1.0
```

File: package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
     "jest-environment-node": "^27.4.4",
     "lint-staged": "^13.0.3",
     "minimist": "^1.2.5",
-    "mocha": "^8.0.1",
```
Comment (RuTsEST) [inline package.json:61]: `json5`, `qs`, `formidable`, `decode-uri-component` and `minimatch` missing

**Review by RuTsEST — APPROVED** [review]

**Comment by pipedrive-bot-dora** [comment]
## Neodora Analysis ✅


Commit [`d18acc1`](https://github.com/pipedrive/client-nodejs/commit/d18acc1c5616c70694577c2f8557cce577e58a2f/)  Job [`136784`](https://jenkins.pipedrive.tools/job/neodora/136784/)

**Pull request merged**